core/server/data/validation/index.js

```diff
@@ -190,7 +190,12 @@ validateSchema = function validateSchema(tableName, model) {
 
         // TODO: check if mandatory values should be enforced
         if (model[columnKey] !== null && model[columnKey] !== undefined) {
-            // check length
+            // check soft limits first, if present (validations: {isLength: {min, max}})
+            if (_.has(schema[tableName][columnKey], 'validations.isLength')) {
```

Comment:
Checking for existing soft limits first, before we actually check for hard limits.

Reply:
That looks a bit complicated, because the `isLength` validator is auto triggered in the `validate` fn.
I think there is an easier implementation possible. We simply check if a specific translation key exists e.g. `notices.data.validation.index.validationFailedTypes.{type}`. (type could be isLength).